src/adera/activescene/VehicleBuilder.h

```diff
@@ -133,11 +135,11 @@ class VehicleBuilder
     void set_prefabs(std::initializer_list<SetPrefab> const& setPrefab);
 
 
-    WeldId weld(osp::ArrayView<PartToWeld const> toWeld);
+    WeldId weld(std::span<PartToWeld const> toWeld);
```

Comment:
I chose Corrade ArrayView over std::span as it features more human-friendly and easier to read methods for making 'subspans'. eg: methods like prefix() and suffix().

ArrayView also supports constructing from std::initializer_list, which is relied upon in a few places here, likely on this line too, which probably causes a build break.

I remember trying to help someone with their C++ project in a voice call one time, explaining std::initializer_list, std::array, and c-style array since they didn't know which one to use. They did not want to do anymore C++ shortly afterwards.

Reply:
Initializer list works with span though?